tests/FSharpPlus.Suave.Tests/FleeceTests.fs

```diff
@@ -0,0 +1,87 @@
+module FSharpPlus.Suave.Tests.FleeceTests
+
+open FSharpPlus.Suave.Tests.Fleece
+open Expecto
+open SuaveTest
+open Suave.Http
+open FSharpPlus
+open FSharpPlus.Data
+
+open Fleece
+open Fleece.FSharpData
+open Fleece.FSharpData.Operators
+open FSharp.Data
+open FSharpPlus.Suave
+open FSharpPlus.Suave.Http
+open FSharpPlus.Suave.Successful
+open FSharpPlus.Suave.Filters
+open FSharpPlus.Suave.RequestErrors
+open FSharpPlus.Suave.Writers
+open FSharpPlus.Operators
+
+type Note={ text:string }
+  with static member create n={text=n}
+       static member OfJson json =
+         match json with
+         | JObject o -> Note.create <!> (o .@ "text")
+         | x -> Failure (sprintf "Expected note, found %A" x)
+
+       static member ToJson (x: Note) =
+         jobj [
+            "text" .= x.text
+         ]
+let sampleNotes=["hej"; "svejs";] |> map Note.create
+let webPart ()=
+  let overview =
+    GET >=> (fun (ctx) ->
+            monad {
+              return! Json.OK sampleNotes ctx
```

Comment:
This is a bit unusual. Normally you won't use a computation expression for one line.
You can write it directly like:

    let overview = GET >=> (fun ctx -> Json.OK sampleNotes ctx)

But unless you want to keep the context explicit you can shorten it to:

    let overview = GET >=> Json.OK sampleNotes

Reply:
Indeed, since it's a test, it doesn't really make sense to write it like this.